src/test/bloom_tests.cpp

```diff
@@ -308,10 +313,10 @@ BOOST_AUTO_TEST_CASE(merkle_block_2_with_update_none)
     for (unsigned int i = 0; i < vMatched.size(); i++)
         BOOST_CHECK(vMatched[i] == merkleBlock.vMatchedTxn[i].second);
 
-    // Match an output from the second transaction (the pubkey for address 1DZTzaBHUDM7T3QvUKBz4qXMRpkg8jsfB5)
-    // This should not match the third transaction though it spends the output matched
-    // It will match the fourth transaction, which has another pay-to-pubkey output to the same address
-    filter.insert(ParseHex("044a656f065871a353f216ca26cef8dde2f03e8c16202d2e8ad769f02032cb86a5eb5e56842e92e19141d60a01928f8dd2c875a390f67c1f6c94cfc617c0ea45af"));
+    // Match an output from the second transaction (the pubkey for address mjuZa8Dy12HKyUNjc1whNTxRaGU4gGGLxY)
+    // It will match the third transaction, which has another pay-to-pubkey output to the same address
+    // This should not match the last transaction though it spends the output matched
```

Comment:
When deserializing the hex string into a block object, the txids are computed the new way so they don't match the txids used in the test.